manifests/charts/base/files/profile-demo.yaml

```diff
@@ -0,0 +1,69 @@
+# The demo profile enables a variety of things to try out Istio in non-production environments.
+# * Lower resource utilization.
+# * Some additional features are enabled by default; especially ones used in some tasks in istio.io.
+# * More ports enabled on the ingress, which is used in some tasks.
+meshConfig:
+  accessLogFile: /dev/stdout
+  extensionProviders:
+    - name: otel
+      envoyOtelAls:
+        service: opentelemetry-collector.istio-system.svc.cluster.local
+        port: 4317
+    - name: skywalking
+      skywalking:
+        service: tracing.istio-system.svc.cluster.local
+        port: 11800
+    - name: otel-tracing
+      opentelemetry:
+        port: 4317
+        service: opentelemetry-collector.otel-collector.svc.cluster.local
```

Comment:
These should be installed as needed

Reply:
This is for otel demos: https://github.com/istio/istio/tree/master/samples/open-telemetry

Reply:
:+1: its a demo profile to install a bunch of stuff for you so you can go through all the istio.io docs. Its a direct copy from the istioctl profile, nothing new -- just adding support for Helm.

Reply:
this's very useful to simply test setup for istio.io.